app/controllers/concerns/shared_methods.rb

```diff
@@ -0,0 +1,13 @@
+module SharedMethods
```

Comment:
So close!! I'm assuming you wanted to use Rails's new ActiveModel concerns? First, awesome job recognizing the opportunity to DRY up your code! 👍🏾I found [a post about using concerns by DHH himself](https://signalvnoise.com/posts/3372-put-chubby-models-on-a-diet-with-concerns), but I found it a bit hard to understand. [This walkthrough](https://richonrails.com/articles/rails-4-code-concerns-in-active-record-models), with a before and after of a model, was easier to digest.